src/main/jbake/content/dukes-forest001.adoc

```diff
@@ -16,13 +16,13 @@ Duke's Forest is a simple e-commerce application that contains several
 web applications and illustrates the use of the following Jakarta EE
 APIs:
 
-* JavaServer Faces technology, including Ajax
-* Contexts and Dependency Injection for Jakarta EE (CDI)
-* Java API for RESTful Web Services (JAX-RS)
-* Java Persistence API (JPA)
-* Java API for JavaBeans Validation (Bean Validation)
-* Enterprise JavaBeans (EJB) technology
-* Java Message Service (JMS)
+* Jakarta Faces technology, including Ajax
+* Jakarta Dependency Injection (CDI)
```

Comment:
Jakarta Contexts and Dependency Injection

Reply:
@antoinesd @starksm64 as CDI spec leads, is it valid to use Jakarta Dependency Injection as short name?

Reply:
[Jakarta Dependency Injection](https://jakarta.ee/specifications/dependency-injection/1.0/) is a different spec than [Jakarta Contexts and Dependency Injection](https://jakarta.ee/specifications/cdi/2.0/).  CDI builds on DI.

Reply:
Ok, tha's why I was so sure I had read Jakarta Dependency Injection somewhere... will take care of it.